Widgets/PrivacyIndicator/Button.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+extension PrivacyIndicator {
+
+class ButtonView: UIButton {
+    override func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        self.setConstrains()
+    }
+} // end class ButtonView
```

Comment:
As it is a nested class please keep indents instead of commen "//end class ButtonView"